mico-core/src/main/java/io/github/ust/mico/core/broker/MicoApplicationBroker.java

```diff
@@ -226,6 +231,24 @@ public MicoApplication addMicoServiceToMicoApplicationByShortNameAndVersion(Stri
         }
     }
 
+    /**
+     * Sets the default environment variables for Kafka enabled MicoServices. See {@link MicoEnvironmentVariable.DefaultEnvironmentVariableKafkaNames}
+     * for a complete list.
+     * @param micoServiceDeploymentInfo The correct service needs to be set already
+     */
+    private void setDefaultEnvironmentVariablesForKafkaEnabledServices(MicoServiceDeploymentInfo micoServiceDeploymentInfo){
+        if(micoServiceDeploymentInfo.getService() == null){
+            throw new IllegalArgumentException("The MicoServiceDeploymentInfo needs a valid MicoService set to check if the service is Kafka enabled");
+        }
+        if(!micoServiceDeploymentInfo.getService().isKafkaEnabled()){
+            return;
+        }
+        log.debug("Adding default environment variables to the Kafka enabled MicoService '{}'", micoServiceDeploymentInfo.getService());
```

Comment:
Only print `shortName` and `version` and not the complete MicoService, because that could be a quite big object (including dependencies ...)